apps/communications/contacts/views/details/js/details_ui.js

```diff
@@ -0,0 +1,463 @@
+'use strict';
+
+/* global ContactsButtons, LazyLoader, ActivityHandler, utils, WebrtcClient */
+/* global Normalizer, ContactPhotoHelper */
+/* global TAG_OPTIONS */
+
+/* exported DetailsUI */
+
+(function(exports) {
+
+  var photoPos = 7;
+  var initMargin = 8;
+  // Scale ratio for different devices
+  var SCALE_RATIO = window.devicePixelRatio || 1;
+
+  var DetailsUI = {
+    init: function() {
+      this._ = navigator.mozL10n.get;
+      this.header = document.querySelector('#details-view-header');
+      this.contactDetails = document.querySelector('#contact-detail');
+      this.listContainer = document.querySelector('#details-list');
+      this.detailsName = document.querySelector('#contact-name-title');
+      this.detailsNameText = document.querySelector('#contact-name-title bdi');
+      this.orgTitle = document.querySelector('#org-title');
+      this.datesTemplate = document.querySelector('#dates-template-\\#i\\#');
+      this.addressesTemplate =
+        document.querySelector('#address-details-template-\\#i\\#');
+      this.duplicateTemplate =
+        document.querySelector('#duplicate-contacts-template');
+      this.editContactButton = document.querySelector('#edit-contact-button');
+      this.cover = document.querySelector('#cover-img');
+      this.detailsInner = document.querySelector('#contact-detail-inner');
+      this.favoriteMessage = document.querySelector('#toggle-favorite');
+      this.notesTemplate =
+        document.querySelector('#note-details-template-\\#i\\#');
+      this.socialTemplate = document.querySelector('#social-template-\\#i\\#');
+
+      this.addListeners();
+    },
+
+    addListeners: function() {
+      this.initPullEffect(this.cover);
+
+      utils.listeners.add({
```

Comment:
Use regular `addEventListener` here. No need to load more JS just to add events.